docs/src/overview.md

```diff
@@ -0,0 +1,109 @@
+	
+ Keywords: Shell elements, displacements, rotations
+ 
+11.	[Nearly Incompressible Hyperelasticity](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/quasi_incompressible_hyperelasticity/)  
+
+ This program combines ideas from the examples Incompressible Elasticity and Hyperelasticity to construct a mixed element solving three-dimensional displacement-pressure equations.
+	
+ Keywords: Non-linear problem, hyperelasticity, finite strain, large deformations, Newton's method, automatic differentiation, coupled problem, mixed elements, displacement-pressure equation, incompressibility
+
+12.	[Computational homogenization](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/computational_homogenization/)  
+
+ A two-dimensional representative volume element (RVE) is loaded in shear. Dirichlet and periodic boundary conditions are applied and the results from the homogenization are compared to the Voigt and Reuss bounds.  
+	
+ Keywords: homogenization, periodic boundary conditions, representative volume element (RVE), microscale, Voigt and Reuss bound
+
+13.	[Stokes flow](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/stokes-flow/)  
+
+ Stokes flow on a quarter circle is solved by applying periodic boundary conditions. The weak and strong form of the problem are discussed including constrains. Mesh generation is done directly via the Gmsh API.  
+	
+ Keywords: Periodic boundary conditions, multiple fields, mean value constraint, mesh generation with Gmsh, coupled problem, weak and strong form
+ 
+14.	[Topology Optimization](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/topology_optimization/)  
+
+ Topology optimization is shown for the bending problem by using a SIMP material model. To avoid numerical instabilities, a regularization scheme requiring the calculation of the Laplacian is imposed, which is done by using the grid topology functionalities.  
+	
+ Keywords: Topology optimization, weak and strong form, non-linear problem, Laplacian, grid topology
+
+## Other examples
+1.	[Incompressible Navier-Stokes Equations via DifferentialEquations.jl](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/ns_vs_diffeq/)  
+
+ The weak form of the semi-discretized incompressible Navier-Stokes equations are derived from the strong form and implemented in the finite element code. Then, the time-dependent solution is calculated by using the DifferentialEquations.jl package. For this purpose, the PDE is required in a specific form.  
+	
+ Keywords: Fluid dynamics, weak and strong form, solver, incompressibility, time-dependent problem
+
+## Grouped by topic
+* Fundamentals: [Heat Equation](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/heat_equation/), [Postprocessing](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/postprocessing/), [Helmholtz Equation](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/helmholtz/)
```

Comment:
```suggestion
* Fundamentals: [Heat Equation](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/heat_equation/), [Postprocessing](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/postprocessing/), [Helmholtz Equation](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/helmholtz/), [Incompressible Elasticity](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/incompressible_elasticity/), [Computational homogenization](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/computational_homogenization/)
```

Reply:
I just thought that all examples should be listed in some category.
That's why I added the ones that were not listed in any category yet.
I think that it would be nice to have an examples under "Fundamentals" which shows how to treat multiple fields.
(Even if it is pretty straight forward.) And "Incompressible Elasticity" would be a good choice for that, I think.

Regarding "Computational Homogenization" you are probably right: Its too specific for "Fundamentals".

Reply:
I agree about the part that every example should be listed at least in one category. If not category is a good fit, we should add a category "other examples".

Personally, I would consider using FEM with multiple fields and coupling to be already too advanced for "Fundamentals". That does not mean it has to be a very hard problem, but most students / beginners should start with simpler problems in my opinion, since there are already a lot of additional steps to consider for coupled problems.

Reply:
Maybe we could have a category "Coupled Problems" and list this example as the first one?

Reply:
I think I will add two categories: "Multiple fields" to account for all problems which are either fully coupled, unidirectional coupled or contain multiple fields for the calculation (shells) and "Periodic problems" for the examples Comp. homogenization and stokes flow. This way, each example is present at least once in the Grouped by topic section.

Reply:
I think we should be a bit careful regarding the wording. I would classify "coupled problems" as a subsection of problems containing multiple fields, where, loosely speaking, different sets of fields have different underlying physics (e.g. fluid-stucture interacton). Incompressibility on the other hand acts as a constraint on your PDE. Certainly, incompressibility is advanced from a purely theoretical perspective, but apart from this is the algorithmic treatment of incompressible linear elasticity rather straight forward. 

However, I agree hat we might want to have a simpler first example on multiple fields that does not have this large theoretical building (for users without background in mechanics). I can set up some reaction-diffusion system as a simple first example for multiple fields here for the future, if this is what we need.

Reply:
```suggestion
* Fundamentals: [Heat Equation](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/heat_equation/), [Postprocessing](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/postprocessing/), [Helmholtz Equation](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/helmholtz/), [Incompressible Elasticity](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/incompressible_elasticity/)
```